CONTRIBUTING.md

```diff
@@ -1,6 +1,29 @@
-Before contributing code, please note:
+# Before reporting a bug
+1. Search the [issue tracker](https://github.com/aframevr/aframe/issues) for similar issues. 
+2. Try the latest dev branch version of [aframe](https://github.com/aframevr/aframe/tree/dev).
+
+# How to report a bug
+1. Specify the revision number of the aframe.js library where the bug occurred.
+2. If you are using a PC, specify your browser version, operating system, and graphics card (for example, Chrome 23.0.1271.95, Windows 7, Nvidia Quadro 2000M).
```

Comment:
For 2/3, maybe just simplify it to `Specify information about your browser and system (e.g., "Firefox Nightly on OS X")`